src/test/e2e-emulated/contracts/repeat-range.tact

```diff
@@ -0,0 +1,41 @@
+contract RepeatRange {
+    /// Effective range counter
+    effCounter: Int = 0;
+
+    /// To handle deployment
+    receive() {}
+
+    /// from -2^{256} to 0, including both ends
+    get fun testIgnoredRange(): Int {
+        let counter = 0;
+
+        repeat ((-pow(2, 255)) * 2) { counter += 1 }
+        repeat (-1) { counter += 1 }
+        repeat (0) { counter += 1 }
+
+        return counter; // 0, if things go right
+    }
+
+    /// from 2^{31} to +∞ — repeat range is too big
+    get fun testInvalidRange(): Int {
+        let caseCounter = 0;
+
+        try {
+            repeat (pow(2, 31)) { caseCounter -= 1 }
+        } catch (_) { caseCounter += 1 }
```

Comment:
Let's check we get a specific exit code here, it's not immediately obvious what `caseCounter += 1` means here.